### DISCUSSION TITLE
Comparison with the original MTGAP layout

### DISCUSSION THREAD

**Discussion by sunaku** [binarybottle/engram-layout#18] [Q&A] [answered]
Hi Arno,

Apparently MTGAP 2.0 is not as widely accepted as [the original MTGAP](https://mathematicalmulticore.wordpress.com/the-keyboard-layout-project/), which is generally revered by layout enthusiasts for its roll-heavy "piano" feel.  This fact seems to have detracted from Engram's layout comparison [as a glaring omission](https://www.reddit.com/r/KeyboardLayouts/comments/ncewbo/arnos_engram_keyboard_layout/gyl95gd?utm_source=share&utm_medium=web2x&context=3).  To rectify this, could you please add the original MTGAP layout (as shown below) to the comparison notebook for the sake of completeness?

```
 Y  P  O  U  J   K  D  L  C  W
 I  N  E  A  ;   M  H  T  S  R
 Q  Z  <  >  ?   B  F  G  V  X

 y  p  o  u  j   k  d  l  c  w
 i  n  e  a  ,   m  h  t  s  r
 q  z  /  .  :   b  f  g  v  x
 ```

The only comparison of the original MTGAP and Engram 2.0 that I know of is this one based on Cryptography bigram frequencies:

https://github.com/bclnr/kb-layout-evaluation/

Thanks.

**Comment by binarybottle** [comment, marked as answer]
Thank you for the Reddit post and pointer -- nice discussion there! 

I tried to restrict my comparison table to layouts that are (1) published online, (2) have a way to install, and (3) are the versions promoted by their authors.  MTGAP 2.0 barely satisfies criteria 1 and 2, but does satisfy criterion 3.

**Comment by binarybottle** [comment]
See https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo!
